float32 positionx
float32 positiony
------
int8 obstacle_probability 